src/app/components/modules/simplified-lunch/simplified-lunch.component.ts

```diff
@@ -0,0 +1,118 @@
+import { Component, Input, OnInit, OnDestroy, ElementRef, ViewChild } from '@angular/core';
+import moment from 'moment';
+import * as ElementQueries from 'css-element-queries/src/ElementQueries';
+
+import { AlertService } from '../../../services/alert.service';
+import { LunchService } from '../../../services/lunch.service';
+import { UserService } from '../../../services/user.service';
+
+
+@Component({
+  selector: 'mymicds-simplified-lunch',
+  templateUrl: './simplified-lunch.component.html',
+  styleUrls: ['./simplified-lunch.component.scss']
+})
+export class SimplifiedLunchComponent implements OnInit {
+
+  	loading = true;
+  	lunchDate = moment();
```

Comment:
This isn't being used anywhere, but should probably be used in line 41 where you simply pass in `moment()` for the current date.